subxt/src/error/dispatch_error.rs

```diff
@@ -59,6 +59,9 @@ pub enum DispatchError {
         "Some resource (e.g. a preimage) is unavailable right now. This might fix itself later."
     )]
     Unavailable,
+    /// Root origin is not allowed.
+    #[error("Root origin is not allowed.")]
+    RootNotAllowed,
```

Comment:
Would be interesting if we could automatically check that all types we have from substrate are in sync.

I'm not aware of a clean way to do that.
We could probably explore in the future a custom crate that compares the tokens from this type to a github link, or crate version.

```rust

#[sync_test(
   git = "https://github.com/paritytech/polkadot-sdk/blob/master/substrate",
   path  = "substrate/primitives/runtime/src/lib.rs"
)
enum SubstrateType {
  ...
}

Reply:
That's an interesting idea!